src/openrct2/world/Park.h

```diff
@@ -109,20 +109,20 @@ extern uint32_t gGuestsInParkHistory[32];
 extern int32_t _guestGenerationProbability;
 extern uint32_t _suggestedGuestMaximum;
 
-void set_forced_park_rating(int32_t rating);
-int32_t get_forced_park_rating();
+void ParkSetForcedRating(int32_t rating);
+int32_t ParkGetForcedRating();
 
-int32_t park_is_open();
-uint32_t park_calculate_size();
+int32_t ParkIsOpen();
+uint32_t ParkCalculateSize();
 
-void update_park_fences(const CoordsXY& coords);
-void update_park_fences_around_tile(const CoordsXY& coords);
+void ParkUpdateFences(const CoordsXY& coords);
+void ParkUpdateFencesAroundTile(const CoordsXY& coords);
 
-uint8_t calculate_guest_initial_happiness(uint8_t percentage);
+uint8_t CalculateGuestInitialHappiness(uint8_t percentage);
```

Comment:
This should probably be:
```suggestion
uint8_t ParkCalculateGuestInitialHappiness(uint8_t percentage);
```
To be in line with all other functions.

Reply:
Ok